js/signin.js

```diff
@@ -0,0 +1,104 @@
+const emailInput = document.querySelector("#email-input");
+const passwordInput = document.querySelector("#password-input");
+const emailErrorMessage = document.querySelector("#email-error-message");
+const passwordErrorMessage = document.querySelector("#password-error-message");
+const loginForm = document.querySelector("#login-form");
+
+const eyeOffButton = document.querySelector(".eye-off");
+const eyeOnButton = document.querySelector(".eye-on");
+
+const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
+
+// 이메일 입력
+emailInput.addEventListener("focusout", checkEmailInput);
+emailInput.addEventListener("focus", resetInputStyle);
+
+function checkEmailInput() {
+  if (emailInput.value.trim() === "") {
+    showEmailErrorMessage("이메일을 입력해 주세요.");
+  } else if (!validateEmail(emailInput.value.trim())) {
+    showEmailErrorMessage("올바른 이메일 주소가 아닙니다.");
+  } else {
+    hideErrorMessage();
+  }
```

Comment:
조건식 내에서 계속 사용되기 때문에 미리 요렇게 변수로 만들어놓고 작성해주셔도 좋을 것 같아요!

```suggestion
  const email = emailInput.value.trim();
  
  if (email === "") {
    showEmailErrorMessage("이메일을 입력해 주세요.");
  } else if (!validateEmail(email)) {
    showEmailErrorMessage("올바른 이메일 주소가 아닙니다.");
  } else {
    hideErrorMessage();
  }
```

Reply:
소소하지만 요렇게 형식을 맞춰주는 것도 좋을 것 같아요!

```suggestion
  if (emailInput.value.trim() === "") {
    showEmailErrorMessage("이메일을 입력해 주세요.");
  } else if (!validateEmail(emailInput.value.trim())) {
    showEmailErrorMessage("올바른 이메일 주소가 아닙니다.");
  } else {
    hideEmailErrorMessage();
  }
```

Reply:
요런 느낌도 괜찮을 것 같아요!

```suggestion
  if (emailInput.value.trim() === "") {
    showEmailErrorMessage("이메일을 입력해 주세요.");
    return;
  }
  
  if (!validateEmail(emailInput.value.trim())) {
    showEmailErrorMessage("올바른 이메일 주소가 아닙니다.");
    return;
  }
  
  hideErrorMessage();
```